<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OBD Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCANMessageButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Send the test CAN Message</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentText">
     <string>COM1</string>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>110</string>
    </property>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>141</y>
      <width>1121</width>
      <height>401</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>34</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <column>
     <property name="text">
      <string>Counter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Std. ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ext. ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IDE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RTR</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DLC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[0]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[1]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[2]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[3]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[4]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[5]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[6]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data[7]</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>FMI</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
